README.md

```diff
@@ -2,103 +2,103 @@
-Initialize your workspace:
+Make sure all required dependecies are installed using
+[`rosdep`](http://wiki.ros.org/rosdep). From the top directory of your
+catkin workspace run (assuming this repository is cloned directly
+under `src`):
 ```
-catkin init
+rosdep update && rosdep install --from-paths src/sd_twizymodel/ --ignore-src -r -y
 ```
 
-#### C. Clone this repository or copy its contents at your `~/catkin_ws/src` folder of the catkin workspace you just initialized.
-#### D. Navigate to your workspace and build the simulation
-```
-cd ~/catkin_ws
-catkin build sd_robot sd_control sd_description
-```
-If you have previously built your workspace with catkin_make:
-Either clean your workspace with `catkin clean` and rebuild with `catkin build`
-or build the SD Twizy Gazebo packages in isolation with `catkin_make --pkg sd_robot sd_control sd_description`.
-After the built has successfully finished, do:
-```
-source devel/setup.bash
-```
-#### E. Launch the simulation:
-This launches the vehicle model in Gazebo and RViz for visualizing the sensors' output.
+Now you can build the packages in this repository. `catkin_make`,
+[`catkin_tools`](https://catkin-tools.readthedocs.io/en/latest/installing.html)
+and
+[`colcon`](https://catkin-tools.readthedocs.io/en/latest/installing.html)
+should all work.
+
+### Autoware
+
+The `sd_autoware` package included in this repository provides an
+interface to be able to control the StreetDrone model using the
+[Autoware.AI](https://catkin-tools.readthedocs.io/en/latest/installing.html)
+open-source self-driving car stack.
+
+By default this package is not built to prevent any problems when
+Autoware.AI is not available. If it is available and you do want to
+use the package, remove the file
+`streetdrone_model/sd_autoware/CATKIN_IGNORE` and build again.
+
+## Launch the simulation
+The following launches the vehicle model in Gazebo and RViz for
+visualizing the sensors' output:
+
 ```
-roslaunch sd_robot sd_twizy_empty.launch
+roslaunch sd_bringup sd_twizy_gazebo.launch
 ```
 
-<p align="center"> 
-<img src="streetdrone_model/sd_docs/imgs/sd.png">
+<p align="center"> <img src="streetdrone_model/sd_docs/imgs/sd.png">
 </p>
 
-You might need to update your ignition-math version `sudo apt upgrade libignition-math2`
+There are several more launch files included to provide a more
+interesting surrounding for the vehicle; try for instance:
 
-##### Display the robot only in Gazebo:
-To only launch the model in Gazebo, do:
 ```
-roslaunch sd_robot sd_twizy_empty_gazebo.launch 
+roslaunch sd_bringup sd_twizy_gazebo_default.launch
 ```
 
-##### Known Issues:
-* On Gazebo7 the model logs an error on the p-gain of the wheels' velocity controllers. This does not affect the performance of the model and has only been detected when running the gazebo-ros-control package for Gazebo7. If you choose to build and run the model in Gazebo8 with gazebo8-ros-control, the problem disappears.
```

Comment:
Please include the following:
##### Known Issues:
- `[Err] [REST.cc:205] Error in REST request.` This is a known error in Gazebo 9. To fix it, change this `~/.ignition/fuel/config.yaml` from `https://api.ignitionfuel.org` to` https://api.ignitionrobotics.org`
- `Couldn't open joystick force feedback!` The error is non-fatal and can be ignored. It's a common error in the joystick_drivers node